Sprintfinity3902/Entities/Enemies_NPCs/HandEnemy.cs

```diff
@@ -9,7 +12,7 @@ namespace Sprintfinity3902.Entities
     public class HandEnemy : AbstractEntity, IEnemy
     {
 
-        private static float F_DOT_TWO = .2f;
+        private static float F_DOT_THREE = .4f;
```

Comment:
Name doesn't match reality anymore. I think we should change our Magic number variables to descriptive names (in this case, something like ENTITY_SPEED) so we don't have to update both the number and the variable name.